src/main/resources/application.properties

```diff
@@ -89,6 +89,9 @@ uv.security.oidc.client-id=
 uv.security.oidc.client-secret=
 uv.security.oidc.logout-uri=
 
+# JSP template engine---------------------------------------------------------------------------------------------------
+uv.template-engine.jsp.use-precompiled=false
```

Comment:
lieber nur in die readme packen und es ist durch die `JspProperties` eh schon false